platform/ui/src/components/Input/Input.jsx

```diff
@@ -1,50 +1,47 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Label from '../Label';
+import classnames from 'classnames';
+
+const baseInputClasses =
+  'shadow transition duration-300 appearance-none border rounded w-full py-2 px-3 text-sm text-gray-700 hover:border-gray-500 leading-tight focus:border-gray-500 focus:outline-none';
+
+const transparentClasses = {
+  true: 'bg-transparent',
+  false: '',
+};
 
 const Input = ({
   label,
   containerClassName = '',
   labelClassName = '',
   className = '',
-  transparent = defaults.transparent,
-  ...rest
+  transparent = true,
+  id,
+  ...otherProps
```

Comment:
Similar to feedback I gave @rodrigolabs earlier, I want to try and avoid spreading props. It's handy, but in larger apps it can make things very confusing / difficult to debug.

Reply:
I've created a new PR adding this eslint rule: https://github.com/OHIF/Viewers/pull/1509